src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/dto/JournalføringRequest.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.tilleggsstonader.sak.journalføring.dto
+
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.kontrakter.journalpost.LogiskVedlegg
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingÅrsak
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvis
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvis
+import java.time.LocalDate
+
+data class JournalføringRequest(
+    val dokumentTitler: Map<String, String>? = null,
+    val logiskeVedlegg: Map<String, List<LogiskVedlegg>>? = null,
+    val ident: String,
+    val stønadstype: Stønadstype,
+    val oppgaveId: String,
+    val journalførendeEnhet: String, // TODO: Hvorfor sendes denne inn?
```

Comment:
Ja bra spm. 
I gosys kan saksbehandler velge hvilken enhet man jobber for, og den enheten brukes vid journalføring. Husker ikke hvordan det ble gjort i EF, men den er vel hardkodet/fra azure-token eller at man bruker den enhet som er tilknyttet type stønad/tema etc ? 

Tror vi burde se litt mer på det der å faktiskt få den til å bli riktig.

Reply:
Løser i egen oppgave